website/labels.php

```diff
@@ -33,7 +33,20 @@
 $opis = strtr(strip_tags($opis), $conaco);
 $id = sprintf('GK%04X', $id);
 
-// ------ języki dostępne -----//
+// ------ available templates -----//
+
+$labelTemplates = '';
+
+//include all templates, needed to see all templates in declared_classes
+foreach (glob('lib/labels/*/*.php') as $filename) {
```

Comment:
:+1: this new mechanism of dynamic mapping is good improvement

Reply:
Actually I was thinking that there should be some better mechanism for this ("get all subclasses of the class"), but I was not able to find anything better than this `glob` and hoping that filenames match classnames...

Reply:
interesting question

following [this SO post](https://stackoverflow.com/questions/6671398/get-all-defined-classes-of-a-parent-class-in-php), I tried a little bit "instanceof" way without success, 
but maybe we could try to follow _registar pattern_ suggested by _Ahmad Hajjar_ ..

Reply:
hum..not sure registar is a good way too .. I dont know

I finally think we would leave as you did